proposals/nnnn-streaming-media-manager-update.md

```diff
@@ -0,0 +1,148 @@
+2. Starting
+	- Streaming sessions are being opened, and encoder is being prepared. Can move to **Ready** only if HMI is **Limited or greater** and Phone State is **Regaining Active** or **Active**. If not, move to **Stopped**.
+3. Ready
+	- Streaming sessions and encoder are ready. This state is only active so long as HMI is **Limited or greater** and Phone State is **Regaining Active** or **Active**. If at this state, and Phone State changes to **Resigning Active**, send ~30 frames. If at this state, and HMI changes to **Background or less**, move to **Stopped**.
+
+###### Video Lifecycle Chart
+HMI State   | App State         | Can Open Video Stream | Should Close Video Stream
+------------|-------------------|-----------------------|--------------------------
+NONE        | Background        | No                    | Yes
+NONE        | Regaining Active  | No                    | Yes
+NONE        | Foreground        | No                    | Yes
+NONE        | Resigning Active  | No                    | Yes
+BACKGROUND  | Background        | No                    | Yes
+BACKGROUND  | Regaining Active  | No                    | Yes
+BACKGROUND  | Foreground        | No                    | Yes
+BACKGROUND  | Resigning Active  | No                    | Yes
+LIMITED     | Background        | No                    | No
+LIMITED     | Regaining Active  | Yes                   | Yes
+LIMITED     | Foreground        | Yes                   | No
+LIMITED     | Resigning Active  | No                    | No
+FULL        | Background        | No                    | No
+FULL        | Regaining Active  | Yes                   | Yes
+FULL        | Foreground        | Yes                   | No
+FULL        | Resigning Active  | No                    | No
+
+###### Audio Stream Lifecycle Chart
+Since the app state is irrelevant for audio streaming (because we can do this in the background), the lifecycle chart is a bit simpler than the video lifecycle chart.
+
+HMI State   | Can Open Audio Stream | Should Close Audio Stream
+------------|-----------------------|--------------------------
+NONE        | No                    | Yes
+BACKGROUND  | No                    | Yes
+LIMITED     | Yes                   | No
+FULL        | Yes                   | No
+
+##### Sending Video Data
+The way in which video frames are sent over will not change from the current implementation. This function will require input of a `CVImageBufferRef` for video. It will be up to the developer to decide how to frequently to call this function. The developer will be able to know if the currently connected stream is encrypted with a `BOOL` `isEncrypted`, a `CGSize` `screenSize` to let them know the current Head Unit screen size, and a `CVPixelBufferPoolRef` for using the encoder’s pixel buffer pool, as is currently available in SMM.
```

Comment:
I assume `isEncrypted` et. al. are actually `encrypted` with the getter prepending the `is`?